kedro/framework/cli/cli.py

```diff
@@ -63,7 +63,7 @@ def info():
     click.echo(
         "kedro allows teams to create analytics\n"
         "projects. It is developed as part of\n"
-        "the Kedro initiative at QuantumBlack."
+        "the Kedro initiative."
```

Comment:
Might I suggest this text becomes: 
"Kedro is a Python framework for\n"
"creating reproducible, maintainable\n"
"and modular data science code."